### PR TITLE
Update per-machine-installation.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Tiny edit, only moved a full stop punctuation mark.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/per-machine-installation.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The per-machine sync app supports syncing OneDrive and SharePoint files in Micro
 
 - All Windows versions supported by the sync app. [Learn more](https://support.office.com/article/cc0cb2b8-f446-445c-9b52-d3c2627d681e)
```
Comment (JohanFreelancer9) [inline OneDrive/per-machine-installation.md:40]:
```suggestion
- All Windows versions supported by the sync app. [Learn more](https://support.office.com/article/cc0cb2b8-f446-445c-9b52-d3c2627d681e).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @baardhermansen, one more minor additional edit, please. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@adeejoseph PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**